lib/actions/down.js

```diff
@@ -8,37 +9,49 @@ const migrationsDir = require("../env/migrationsDir");
 const hasCallback = require('../utils/has-callback');
 
 module.exports = async (db, client) => {
+  const isBlockRollback = _.get(global.options, "block");
   const downgraded = [];
   const statusItems = await status(db);
   const appliedItems = statusItems.filter(item => item.appliedAt !== "PENDING");
   const lastAppliedItem = _.last(appliedItems);
+  
+  let itemsToRollback = [];
```

Comment:
To avoid mutability

```suggestion
    const itemsToRollback = isBlockRollback && lastAppliedItem.migrationBlock ? appliedItems.filter(item => item.migrationBlock === lastAppliedItem.migrationBlock).reverse() : [lastAppliedItem]
```

Reply:
I will take this into account in a next PR if it's ok for you as I will make a proposal of improvement for up & down commands.

Reply:
Here it is. I will modify the base branch once this one will be merged into master. 
https://github.com/ftreguer/migrate-mongo/pull/1